drivers/ieee802154/ieee802154_b9x_frame.c

```diff
@@ -607,8 +608,8 @@ b9x_ieee802154_get_data(const uint8_t *payload,
 #if CONFIG_SOC_RISCV_TELINK_B91 || CONFIG_SOC_RISCV_TELINK_B92
 #include <aes.h>
 #elif CONFIG_SOC_RISCV_TELINK_B95
-#include <ske.h>
-#include <ske_portable.h>
+#include <lib/include/ske/ske.h>
+#include <lib/include/ske/ske_portable.h>
```

Comment:
same "lib/include/ske/" can be removed after move into CMake

Reply:
will add the lib/include /ske into cmake list .